### PR TITLE
Overall code cleanup

### PR DESCRIPTION
**Pull request by tarcisiofischer**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,4 +4,6 @@ bin/
 *.o
 .vscode/
 *.dll
-*.exe
\ No newline at end of file
+*.exe
+/cmake-build-release/
```
Comment (648trindade) [inline .gitignore]: using CLion? What do you think about placing such folders inside a `build` directory at project root? This way we don't need to create an entry here to every cmake profile

File: src/c++/simulator/geometry.hpp [hunk 2]
```diff
@@ -97,9 +97,10 @@ struct Edge : public PrimitiveGeometry
 struct PlanarGeometry : public PrimitiveGeometry 
 {
 public:
-	Vector get_normal();
-	Vector get_unitary_normal();
-	double get_volume() override;
+    Vector get_normal() const;
+    Vector get_unitary_normal() const;
+	double get_volume() const override;
```
Comment (648trindade) [inline src/c++/simulator/geometry.hpp]: difference in indentation
Do we have a .clang-format file? we can define one

**Review by 648trindade — APPROVED** [review]

File: src/c++/simulator/geometry.hpp (hunk 2 above)
Comment (tarcisiofischer) [reply src/c++/simulator/geometry.hpp]: I believe we don't have one. What clang-format default should I use? hehehe

File: .gitignore (hunk 1 above)
Comment (tarcisiofischer) [reply .gitignore]: Ah, yes. True. Will try to do it

**Pull request merged**